code/game/turfs/turf.dm

```diff
@@ -79,12 +79,14 @@
 		spawn( 0 )
 			src.Entered(AM)
 			return
-	turfs |= src
```

Comment:
That's largely because of ChangeTurf(), I believe---I do think there's a few rare exceptions where new turfs are explicitly created.